idm_22.2.1_release (#2348)
oracle/docker-images: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

> Building an Oracle Unified Directory image with WebLogic Image Tool
> ===================================================================

Reviewer: ```suggestion
# Building an Oracle Unified Directory image with WebLogic Image Tool
```

Author: changed

> # Introduction

Reviewer: ```suggestion
## Documentation
```

Author: changed

> For instructions on building an OUD image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oud/create-or-update-image#create-or-update-an-oracle-unified-directory-image-using-the-weblogic-image-tool).

Reviewer: ```suggestion
Detailed instructions on how to [create or update an Oracle Unified Directory container image](https://oracle.github.io/fmw-kubernetes/oud/create-or-update-image#create-or-update-an-oracle-unified-directory-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the URL to create-or-update-image does not exist and there does not appear to be any open PR that would create it.

> # Licensing & Copyright

Reviewer: ```suggestion
## License
```

Author: changed

> # Building an Oracle Unified Directory image with WebLogic Image Tool
> ===================================================================

Reviewer: ```suggestion
# Building an Oracle Unified Directory image with WebLogic Image Tool
```
